### PR TITLE
CVE and linting fixes

### PR DESCRIPTION
**Pull request by simps614**
Fixes `rails-html-sanitizer` CVE and standard fixes.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -195,9 +195,9 @@ GEM
       activesupport (>= 5.0.0)
       minitest
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.6.0)
+    rails-html-sanitizer (1.6.2)
       loofah (~> 2.21)
-      nokogiri (~> 1.14)
+      nokogiri (>= 1.15.7, != 1.16.7, != 1.16.6, != 1.16.5, != 1.16.4, != 1.16.3, != 1.16.2, != 1.16.1, != 1.16.0.rc1, != 1.16.0)
```
Comment (morse255) [inline Gemfile.lock:200]: 🤯 😆 Was not expecting this, but 👍.

**Review by morse255 — APPROVED** [review]
LGTM! 👍

File: Gemfile.lock (hunk 1 above)
Comment (simps614) [reply Gemfile.lock:200]: I know it's odd, I can try to change it if you think I should! It happened when I needed to directly upgrade loofah.

Comment (morse255) [reply Gemfile.lock:200]: Haha. No, I think it's fine and is expected behavior. I just wasn't expecting such a fine grained dependency declaration.

**Pull request merged**